api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -126,6 +126,10 @@ DbUser getDbUser() {
   @BeforeEach
   public void setUp() {
     providedWorkbenchConfig = WorkbenchConfig.createEmptyConfig();
+    providedWorkbenchConfig.access.enableAccessRenewal = true;
```

Comment:
many tests need these, so they're now the default.

Please confirm that the deleted lines below correspond only to setting these values to true.